### PR TITLE
All wallets transactions list

### PR DESCRIPTION
**Pull request by Kamill90**

### CODE REVIEW

File: src/screens/Dashboard/TransactionList.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  label: string;
+}
+
+export class TransactionList extends Component<Props> {
+  state = {
+    transactions: [],
+  };
+
+  static getDerivedStateFromProps(props: Props) {
+    const groupedTransactions = [] as any;
```
Comment (Kamill90) [inline src/screens/Dashboard/TransactionList.tsx:19]: If you know cleaner way for this transformation - go ahead

File: loc/en.js [hunk 2]
```diff
@@ -95,7 +95,7 @@ module.exports = {
       tabBarLabel: 'Transactions',
       title: 'transactions',
       description: 'A list of ingoing or outgoing transactions of your wallets',
-      conf: 'conf',
+      conf: 'Confiramtions',
```
Comment (bklukaczewski) [inline loc/en.js:98]: typo

File: android/app/build.gradle [hunk 3]
```diff
@@ -114,7 +114,7 @@ android {
     }
 
     defaultConfig {
-        applicationId "io.goldwallet.wallet"
+        applicationId "com.stxnext.goldwallet"
```
Comment (bklukaczewski) [inline android/app/build.gradle:117]: thanks Kamil! :)

File: src/screens/Dashboard/DashboardScreen.tsx [hunk 4]
```diff
@@ -27,11 +28,16 @@ type Props = NavigationInjectedProps;
 
 export class DashboardScreen extends Component<Props, State> {
   constructor(props: Props) {
-    super(props);
+    super(props);   
+  
+    const allWalletsBalance = BlueApp.getBalance()
+    const AllWallets = BlueApp.getWallets()
+    const wallets = AllWallets.length >1 ? [{ label: 'All wallets', balance: allWalletsBalance, preferredBalanceUnit: 'BTCV' }, ...AllWallets] : AllWallets
```
Comment (bklukaczewski) [inline src/screens/Dashboard/DashboardScreen.tsx:35]: There's something wrong with prettier config as `AllWallets.length >1` looks quite ugly without spaces

File: src/screens/Dashboard/TransactionList.tsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    const dataToGroup = props.data.map((transaction: any) => ({
+      ...transaction,
+      day: moment.unix(transaction.time).format('ll'),
+      walletLabel: transaction.walletLabel || props.label,
+    }));
+    const uniqueValues = [...new Set(dataToGroup.map((item: any) => item.day))];
```
Comment (bklukaczewski) [inline src/screens/Dashboard/TransactionList.tsx:25]: nice 💪

**Pull request merged**